github_activity/github_activity.py

```diff
@@ -530,8 +531,6 @@ def _get_datetime_from_git_ref(org, repo, ref):
 
 def _get_latest_tag(org, repo):
     """Return the latest tag name for a given repository."""
-    response = requests.get(f"https://api.github.com/repos/{org}/{repo}/git/refs/tags")
-    response.raise_for_status()
-    tags = response.json()
-    latest_tag = list(tags)[-1]
-    return latest_tag["ref"].split("/tags/")[-1]
+    out = run("git describe --tags".split(), stdout=PIPE)
+    tag = out.stdout.decode().rsplit('-', 2)[0]
+    return tag
```

Comment:
This will only be the same if the local repo is updated, but I think it's fine and may make sense to check using the local repo's latest tag rather than remote repos anyhow.

Reply:
yeah the other challenge I realized is that using the GitHub API here doesn't sort by latest updated, it sorts alpha-numerically on the tag name. So this will make it more likely we get the latest tag